Integrations/HelloWorld/HelloWorld.py

```diff
@@ -0,0 +1,216 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+''' IMPORTS '''
+
+import json
+import requests
+import dateparser
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' CONSTANTS '''
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API, should not contain Demisto logic.
+    Should do requests and return data
+    """
+
+    def say_hello(self, name):
+        return f'Hello {name}'
+
+    def say_hello_http_request(self, name):
+        """
+        initiates a http request to test url
+        """
+        data = self._http_request(
+            method='GET',
+            url_suffix='/hello/' + name
+        )
+        return data.get('result')
+
+    def list_incidents(self):
+        """
+        returns dummy incident data, just for the example.
+        """
+        return [
+            {
+                'incident_id': 1,
+                'description': 'Hello incident 1',
+                'created_time': datetime.utcnow().strftime(DATE_FORMAT)
+            },
+            {
+                'incident_id': 2,
+                'description': 'Hello incident 2',
+                'created_time': datetime.utcnow().strftime(DATE_FORMAT)
+            }
+        ]
+
+
+def test_module(client):
+    """
+    Returning 'ok' indicates that the integration works like it suppose to. Connection to the service is successful.
+
+    Args:
+        client: HelloWorld client
+
+    Returns:
+        'ok' if test passed, anything else will fail the test
+    """
+
+    result = client.say_hello('DBot')
+    if 'Hello DBot' == result:
+        return 'ok'
+    else:
+        return 'Test failed because ......'
+
+
+def say_hello_command(client, args):
+    """
+    Returns Hello {somename}
+
+    Args:
+        client: HelloWorld client
+        args: all command arguments
+
+    Returns:
+        Hello {someone}
+
+        readable_output: This will be presented in Warroom - should be in markdown syntax - human readable
+        outputs: Dictionary/JSON - saved in incident context in order to be used as input for other tasks in the
+                 playbook
+        raw_response: Used for debugging/troubleshooting purposes - will be shown only if the command executed with
+                      raw-response=true
```

Comment:
```suggestion
      readable_output (str): This will be presented in Warroom - should be in markdown syntax - human readable
        outputs (dict): Dictionary/JSON - saved in incident context in order to be used as input for other tasks in the
                 playbook
        raw_response (dict): Used for debugging/troubleshooting purposes - will be shown only if the command executed with
                      raw-response=true
```